keep/api/arq_worker.py

```diff
@@ -85,9 +125,10 @@ class WorkerSettings:
         conn_retries=10,
         conn_retry_delay=10,
     )
+    # Only if it's an AI-dedicated worker, we can set large timeout, otherwise keeping low to avoid clogging
+    timeout = 60 * 15 if ARQ_TASK_POOL_TO_EXECUTE == ARQ_TASK_POOL_TO_EXECUTE_AI else 30
```

Comment:
@talboren addressing your concern here. If only-ai worker, larger timeouts will be enabled. Otherwise, low timeouts.